.github/workflows/node.js.yml

```diff
@@ -10,20 +10,27 @@ on:
     branches: [ ] # Test on push to every branch
 
 jobs:
-  build:
-
+  maintenance_lts:
+    name: "Maintenance LTS"
```

Comment:
Looking at https://nodejs.org/en/about/releases/ I see there often be two versions in maintenance, and short windows of two active. But I guess we'll test the oldest and newest, and I think this naming is better than trying to be more pedantic about it.